src/librustc_mir/dataflow/mod.rs

```diff
@@ -877,13 +885,18 @@ impl<'a, 'tcx: 'a, D> DataflowAnalysis<'a, 'tcx, D> where D: BitDenotation
     fn propagate_bits_into_entry_set_for(&mut self,
                                          in_out: &IdxSet<D::Idx>,
                                          changed: &mut bool,
-                                         bb: &mir::BasicBlock) {
+                                         bb: &mir::BasicBlock,
```

Comment:
pre-existing, but I think we should change the type of `bb` here to `bb: mir::BasicBlock` -- the `&` is silly

Reply:
fixed !